pages/interview-process.md

```diff
@@ -3,55 +3,39 @@ permalink: /interview-process/
 title: Interview process
 ---
 
-### Phone screen
+### Application review
 
-The first step in the interview process is the phone screen, a 30-minute phone call. Plan to talk about your skills and experience, what you're passionate about, and the work we do here.
+The first step of the process will be a review of the information that you have submitted as part of your application. Your application will be evaluated against the Qualification Competencies detailed below.
 
-### In-person (or videochat) interviews
-
-After your phone interview, the 18F member you spoke to will submit their notes to our Talent Team. If we'd like to chat with you more, we'll ask you to come in to one of our office locations or join us somewhere on the internet for a couple of longer interviews. Depending on what position you're applying for, the format will vary, but generally we group interviews with multiple candidates together into bi-weekly hiring sprints, every other Friday. 
-
-### Basic rubric and criteria
-
-Interviewers evaluate candidates using the basic rubric below, without discussion or input from interview partners or other team members. After they complete a written evaluation, all interviewers meet and discuss the candidate. 
-
-- Principles alignment
-
-	 - Strong desire to serve
-	 - Embraces motivations of open source 
-	 - Embraces user feedback as a way of driving a project 
-	 - Understands working iteratively or as agile team 	
-
-- Technical competence
-
-	 - Knowledge of programming language(s) 
-	 - Knowledge of web framework(s) or other relevant frameworks
-	 - Demonstrated ability to write well-structured code
-	 - Understanding of web technology 
-	 - Understanding of modularity
+### Preliminary screening
 
--  Communication and collaboration skills
+The interview process begins with a preliminary screening, a 30-minute phone or video call. Plan to talk about your skills and experience, what you're passionate about, and the work we do here.
 
-	 - Ability to articulate technical concepts 
-	 - Ability to speak concisely and boldly
-	 - Understands working iteratively or as agile team
-	 - Teachability
-	 - Quick learning
-	 - Flexibility of thought
-
-- Leadership and entrepreneurship
-
-	 - Capable of evangelism
-	 - Can they teach us something we don’t know?
-	 - Will they “raise the bar” of our team?
-
--  Creativity
+### In-person (or videochat) interviews
 
-	 - Ability to translate creativity into practical solutions 
-	 - Change hearts and minds
+After your phone interview, the 18F member you spoke to will submit their notes to our Talent Team. If we'd like to chat with you more, we'll ask you to come in to one of our office locations or join us somewhere on the internet for a couple of longer interviews.
+
```

Comment:
@fureigh I made the following change here:

I'd like to pull this whole sentence out. Technically they aren't applying for a position and I don't think that we will be following the two week sprint interview cycle, at least initially. Too many unknowns with the new process.

"Depending on what position you're applying for, the format will vary, but generally we group interviews with multiple candidates together into bi-weekly hiring sprints, every other Friday."